e2e/testcases/apiservice_test.go

```diff
@@ -36,24 +35,12 @@ const (
 func TestCreateAPIServiceAndEndpointInTheSameCommit(t *testing.T) {
 	nt := nomostest.New(t, nomostesting.Reconciliation1, ntopts.Unstructured, ntopts.RequireGKE(t))
```

Comment:
@tiffanny29631 Remind me why do we require this test to only run on GKE cluster?